modules/text_server_adv/text_server_adv.cpp

```diff
@@ -60,23 +60,17 @@ using namespace godot;
 // Built-in ICU data.
 
 #ifdef ICU_STATIC_DATA
-#include "icudata.gen.h"
+#include <icudata.gen.h>
 #endif
 
 // Thirdparty headers.
 
 #ifdef MODULE_MSDFGEN_ENABLED
-#ifdef _MSC_VER
-#pragma warning(disable : 4458)
```

Comment:
This might cause warnings to be raised by the GDExtension build option for text servers, CC @bruvzg.

We might want to add the same system to godot-cpp.

Reply:
It likely will, but `godot-cpp` does not use `Werror` in any config, so it won't break it. But it definitely worth adding the support for the same system to `godot-cpp` regardless.